pkg/ipamd/ipamd.go

```diff
@@ -1936,7 +1880,7 @@ func (c *IPAMContext) GetPod(podName, namespace string) (*corev1.Pod, error) {
 	}
 	err := c.rawK8SClient.Get(ctx, podKey, &pod)
 	if err != nil {
-		return nil, fmt.Errorf("Error while trying to retrieve Pod Info: %s", err)
+		return nil, fmt.Errorf("Error while trying to retrieve Pod Info: %s", err.Error())
```

Comment:
nit: `%v", err`

Reply:
Why should we log the entire object instead of just error string for information?

Reply:
Good point, this is fine